csharp/ql/test/resources/stubs/Antlr3.Runtime/3.5.1/Antlr3.Runtime.cs

```diff
@@ -1,46 +1,48 @@
 // This file contains auto-generated code.
-
+// Generated from `Antlr3.Runtime, Version=3.5.0.2, Culture=neutral, PublicKeyToken=eb42632606e9261f`.
 namespace Antlr
 {
     namespace Runtime
     {
-        // Generated from `Antlr.Runtime.ANTLRInputStream` in `Antlr3.Runtime, Version=3.5.0.2, Culture=neutral, PublicKeyToken=eb42632606e9261f`
         public class ANTLRInputStream : Antlr.Runtime.ANTLRReaderStream
         {
-            public ANTLRInputStream(System.IO.Stream input, int size, int readBufferSize, System.Text.Encoding encoding) : base(default(System.IO.TextReader)) => throw null;
-            public ANTLRInputStream(System.IO.Stream input, int size, System.Text.Encoding encoding) : base(default(System.IO.TextReader)) => throw null;
+            public ANTLRInputStream(System.IO.Stream input) : base(default(System.IO.TextReader)) => throw null;
             public ANTLRInputStream(System.IO.Stream input, int size) : base(default(System.IO.TextReader)) => throw null;
             public ANTLRInputStream(System.IO.Stream input, System.Text.Encoding encoding) : base(default(System.IO.TextReader)) => throw null;
-            public ANTLRInputStream(System.IO.Stream input) : base(default(System.IO.TextReader)) => throw null;
+            public ANTLRInputStream(System.IO.Stream input, int size, System.Text.Encoding encoding) : base(default(System.IO.TextReader)) => throw null;
+            public ANTLRInputStream(System.IO.Stream input, int size, int readBufferSize, System.Text.Encoding encoding) : base(default(System.IO.TextReader)) => throw null;
         }
-
-        // Generated from `Antlr.Runtime.ANTLRReaderStream` in `Antlr3.Runtime, Version=3.5.0.2, Culture=neutral, PublicKeyToken=eb42632606e9261f`
         public class ANTLRReaderStream : Antlr.Runtime.ANTLRStringStream
         {
-            public ANTLRReaderStream(System.IO.TextReader r, int size, int readChunkSize) => throw null;
-            public ANTLRReaderStream(System.IO.TextReader r, int size) => throw null;
             public ANTLRReaderStream(System.IO.TextReader r) => throw null;
-            public const int InitialBufferSize = default;
+            public ANTLRReaderStream(System.IO.TextReader r, int size) => throw null;
+            public ANTLRReaderStream(System.IO.TextReader r, int size, int readChunkSize) => throw null;
+            public static int InitialBufferSize;
             public virtual void Load(System.IO.TextReader r, int size, int readChunkSize) => throw null;
-            public const int ReadBufferSize = default;
+            public static int ReadBufferSize;
```

Comment:
`const` fields became `static`.

Reply:
Does the stub just need to be updated or does the issue still persist?